RELEASING.md

```diff
@@ -0,0 +1,22 @@
+Release Guide
+-------------
+
+1. Ensure that pulpcore of the corresponding version is available on PyPI.
+1. Generate changelog (`towncrier --yes --version 3.8.0`).
+1. Update version in the `galaxy.yml` and in the `roles/pulp_common/defaults/main.yml`.
+1. Update references to current and previous versions in `docs/index.md`
+   (`sed -i -e 's/3.7.1/3.8.0/g' -e 's/3.6.z/3.7.z/g' -e 's/3.5.z/3.6.z/g' docs/index.md`).
+1. Create a PR with all the changes above and merge it after a review.
+1. Tag this commit with the name of the release, e.g. 3.8.0.
+1. Create a release on github based on the tag.
+1. Upload the pulp_installer collection the Galaxy.
+
+
+Uploading a collection to the Galaxy
+====================================
+
+1. Prepare a tarball of the pulp_insaller collection (from inside the root of the collection, run `ansible-galaxy collection build`).
```

Comment:
There's an ansible version requirement. Either 2.9.x or 2.10.x currently I think. If your distros's RPM is too old, releasers must install from PyPI (in a venv.)

@mdellweg What is our minimum version?

Reply:
with ansible 2.8 you'd need to use `mazer`, but i'd not recommend using 2.8 for anything collection related.
So go with 2.9. You can always install it in a venv.